jujugui/static/gui/src/app/components/env-size-display/env-size-display.js

```diff
@@ -16,8 +16,8 @@ class EnvSizeDisplay extends React.Component {
     const view = e.currentTarget.dataset.view;
     this.props.appState.changeState({
       gui: {
-        machines: view === 'machine' ? '' : null,
-        status: view === 'status' ? '' : null
+        machines: view === 'machines' ? true : null,
```

Comment:
As discussed offline, these will have to be reverted back to their empty string counterparts to avoid issues with state elsewhere in the app where it is expecting a string.

Reply:
While you're at it though, might as well add a comment to this effect so we don't run into it again in the future :)